com.unity.render-pipelines.core/Runtime/Lights/LightAnchor.cs

```diff
@@ -0,0 +1,229 @@
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+using UnityEngine;
+
+namespace UnityEngine
+{
+    /// <summary>
+    /// Represents camera-space light controls around a virtual pivot point.
+    /// </summary>
+    [AddComponentMenu("Rendering/Light Anchor")]
+    [RequireComponent(typeof(Light))]
+    [ExecuteInEditMode]
+    [DisallowMultipleComponent]
+    public class LightAnchor : MonoBehaviour
+    {
+        const float k_ArcRadius = 5;
+        const float k_AxisLength = 10;
+
+        [SerializeField]
+        float m_Distance = 3;
+        [SerializeField]
+        UpDirection m_FrameSpace = UpDirection.World;
+
+        float m_Yaw;
+        float m_Pitch;
+        float m_Roll;
+
+        /// <summary>
+        /// The camera-relative yaw.
+        /// </summary>
+        /// <remarks>
+        /// The range is -180 through 180 inclusive. Values between 0 and 180 are to the right of the camera, and values between 0 and -180 to the left.
+        /// </remarks>
+        public float yaw
+        {
+            get { return m_Yaw; }
+            set { m_Yaw = NormalizeAngleDegree(value); }
+        }
+
+        /// <summary>
+        /// The pitch relative to the horizon or camera depending on value of m_Space.
+        /// </summary>
+        /// <remarks>
+        /// The range is -180 through 180 inclusive. Values between 0 and 180 are below the camera, and values between 0 and -180 are above the camera.
+        /// </remarks>
+        public float pitch
+        {
+            get { return m_Pitch; }
+            set { m_Pitch = NormalizeAngleDegree(value); }
+        }
+
+        /// <summary>
+        /// The camera-relative roll.
+        /// </summary>
+        /// <remarks>
+        /// The range is -180 through 180 inclusive. Values between 0 and 180 are to the right of the camera, and values between 0 and -180 are to the left of the camera.
+        /// </remarks>
+        public float roll
+        {
+            get { return m_Roll; }
+            set { m_Roll = NormalizeAngleDegree(value); }
+        }
+
+        /// <summary>
+        /// The distance from the light's anchor point.
+        /// </summary>
+        public float distance
+        {
+            get { return m_Distance; }
+            set { m_Distance = Mathf.Max(value, .01f); }
+        }
+
+        /// <summary>
+        /// Enum to describes to up vector for the Light Anchor
+        /// </summary>
+        public enum UpDirection
+        {
+            /// <summary>
+            /// Up vector is world space Vector3.up
+            /// </summary>
+            World = Space.World,
+            /// <summary>
+            /// Up vector is the up of the main camera
+            /// </summary>
+            Local = Space.Self
+        }
```

Comment:
This is basically a clone of Space enum. Does it worth it than using Space directly?

Reply:
It's a request from Lighting team for UI purpose on the dropdown.
Show the strings "World" & "Local" instead of "World" & "Self" for clarity.